clusterctl/README.md

```diff
@@ -0,0 +1,123 @@
+## Deploy a mgmt and a workload cluster with clusterctl in kind
+
+### This README is based on docs:
+* https://cluster-api.sigs.k8s.io/clusterctl/overview.html
+* https://github.com/Nordix/airship-clusterapi-docs/blob/clusterctl-kashif/clusterCtl-study/testcmd/instrusctions.md
+  * note files and details of high importance e.g **clusterctl-settings.json**
+
+### create a cluster on localhost
+
+```sh
+~$ kind create cluster
+```
+
+### get kubeconfig of new kind cluster
+
+```sh
+~/.cluster-api$ kind get kubeconfig > kubeconfig
+```
+
+### copy files for cluster-template.yaml and metadata.yaml
+
+```sh
+cp ~/work/go/src/sigs.k8s.io/cluster-api-provider-metal3/examples/clusterctl-templates/clusterctl-cluster.yaml ~/.cluster-api/overrides/infrastructure-metal3/v0.3.0/cluster-template.yaml
```

Comment:
this directory (.cluster-api/overrides/...) will only exist once you have done the local override. You seem to have it a bit later in the workflow. You have to take this section there I suppose.

Reply:
done